portfolio/src/main/webapp/index.html

```diff
@@ -6,10 +6,11 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="getDataMessage()">
```

Comment:
Although "get" is not wrong, I think "loadGreetingsContainer" or something like that might be better.

Reply:
okay I'll make this change